sites/main-site/src/content/blog/2025/survey-results-2025.mdx

```diff
@@ -0,0 +1,324 @@
+subtitle: "Results summary from the nf-core community survey 2025 "
+pubDate: 2025-03-26T12:00:00+01:00
+headerImage: "/assets/images/blog/survey-results-2025/nfcore-survey_logo.png"
+headerImageAlt: "nf-core survey 2025 logo"
+embedHeaderImage: true
+authors:
+    - "jfy133"
+    - "FranBonath"
+label:
+    - "community"
+---
+
+import worldmap from "@assets/images/blog/survey-results-2025/nfcoresurvey2025-worldmap.png";
+import locationdistribution from "@assets/images/blog/survey-results-2025/nfcoresurvey2025-locationdistribution.png";
+import respondertype from "@assets/images/blog/survey-results-2025/nfcoresurvey2025-respondertype.png";
+import confidence from "@assets/images/blog/survey-results-2025/nfcoresurvey2025-confidence.png";
+import happiness from "@assets/images/blog/survey-results-2025/nfcoresurvey2025-happiness.png";
+import cloudglobalimprove from "@assets/images/blog/survey-results-2025/nfcoresurvey2025-cloudglobalimprove.png";
+import cloudglobalpositive from "@assets/images/blog/survey-results-2025/nfcoresurvey2025-cloudglobalpositive.png";
+import cloudglobalrequests from "@assets/images/blog/survey-results-2025/nfcoresurvey2025-cloudglobalrequests.png";
+import barglobaldetails from "@assets/images/blog/survey-results-2025/nfcoresurvey2025-barglobaldetails.png";
+
+import fineuserpositive from "@assets/images/blog/survey-results-2025/nfcoresurvey2025-fineuserpositive.png";
+import fineuserimprove from "@assets/images/blog/survey-results-2025/nfcoresurvey2025-fineuserimprove.png";
+import fineuserrequest from "@assets/images/blog/survey-results-2025/nfcoresurvey2025-fineuserrequest.png";
+import finedevpositive from "@assets/images/blog/survey-results-2025/nfcoresurvey2025-finedevpositive.png";
+import finedevimprove from "@assets/images/blog/survey-results-2025/nfcoresurvey2025-finedevimprove.png";
+import finedevrequest from "@assets/images/blog/survey-results-2025/nfcoresurvey2025-finedevrequest.png";
+
+import { Image } from "astro:assets";
+
+## Introduction
+
+nf-core has a huge community of 11,000 members (based on Slack users), and this number does not seem appear to be diminishing.
+As the community gets bigger, it is getting harder to keep track of all the discussions happening across all of our Slack channels, and increasingly in face-to-face talks on Slack huddles and gather.town.
+Therefore the core team has decided to hold a yearly community survey to help provide us with an overview of the mood within the community and in which ways the 'wind is blowing' in terms of needs and requirements.
+
+With this information, we plan to develop a roadmap for the continued development and progress of the nf-core ecosystem and community over the next few years.
+
+## Key points
+
+- We had 209 responders, representing 1.8% of our Slack community.
+- nf-core has a responder NPS score of 54, representing a very positive opinion of nf-core, and are likely to recommend the community to colleagues and peers.
```

Comment:
First appearance of the acronym, also I saw @FranBonath comment and your response @jfy133 but maybe we could add a pointer to the explanation below like "explained below"
```suggestion
- nf-core has a responder Net Promoter Score (NPS) score of 54, representing a very positive opinion of nf-core, and are likely to recommend the community to colleagues and peers.
```